lib/constants/authentication_type.js

```diff
@@ -0,0 +1,8 @@
+"use strict";

```

Comment:
The master branch should only contain the source code but not the `lib/` folder. Please remove the `lib/` and its sub-directories.